plugins/VanillaStats/js/vanillastats.js

```diff
@@ -753,6 +753,10 @@ var vanillaStats = (function() {
     };
 
     VanillaStats.prototype.getSummaries = function(container) {
+        if (gdn.definition('DashboardSummaries')) {
```

Comment:
Is this correct? It seems like it should be the opposite. Also, you want gdn.getMeta().